lib/secret-storage/nondumpable-allocator.c

```diff
@@ -116,11 +116,20 @@ nondumpable_buffer_alloc(gsize len)
   return buffer->user_data;
 }
 
+static void
```

Comment:
We need to expose this function. Users (modules) cannot use memset for the same reason.

Reply:
Right, wait for the merge please.